packages/utilities/psammead-calendars/src/calendars/jalaali.js

```diff
@@ -0,0 +1,46 @@
+const jalaali = require('jalaali-js');
+
+const jalaaliMonths = {
+  fa: [
+    'فروردین',
+    'اردیبهشت',
+    'خرداد',
+    'تیر',
+    'مرداد',
+    'شهریور',
+    'مهر',
+    'آبان',
+    'آذر',
+    'دی',
+    'بهمن',
+    'اسفند',
+  ],
+  ps: [
+    'وری',
+    'غویی',
+    'غبرګولی',
+    'چنګاښ',
+    'زمری',
+    'وږی',
+    'تله',
+    'لړم',
+    'لیندۍ',
+    'مرغومی',
+    'سلواغه',
+    'کب',
+  ],
+};
+
+function getJalaaliCalendar(gregorianMoment) {
```

Comment:
I was thinking maybe something like this as an API to make the calling code readable and allow other calendars to be added later.

```javascript
import moment from 'moment';
import { jalaali } from '@bbc/psammead-calendars';

const gregorianMoment = moment(someTimestamp);
const gregorianStr = gregorianMoment.locale(locale).format('D MMMM YYYY');
const jalaaliStr = jalaali.formatDate(gregorianMoment);
```

Reply:
If we did
```js
import moment from 'moment';

const gregorianMoment = moment(someTimestamp);
const gregorianStr = gregorianMoment.locale(locale).format('D MMMM YYYY');
const secondaryStr = secondaryCalender && secondaryCalender.formatDate(gregorianMoment);
```

We could pass `secondaryCalender` from the service config to aid unwanted weight on other services :)